sdk/keyvault/CONTRIBUTING.md

```diff
@@ -0,0 +1,84 @@
+# Contributing
+
+Thank you for your interest in contributing to the Azure Key Vault client libraries. As an open source effort, we're excited to welcome feedback and contributions from the community. A great first step in sharing your thoughts and understanding where help is needed would be to take a look at the [open issues][open_issues].
+
+Most contributions require you to agree to a Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us the rights to use your contribution. For details, visit <https://cla.microsoft.com>.
+
+When you submit a pull request, a CLA-bot will automatically determine whether you need to provide a CLA and decorate the PR appropriately (e.g., label, comment). Simply follow the instructions provided by the bot. You will only need to do this once across all repos using our CLA.
```

Comment:
This text is also in each of the library README.md files after the link to this file. Is it intentional that it exists in both places?

Reply:
As I understand it, from a legal standpoint, yes. We'd especially want it in the READMEs that are contained within the packages while this "root" CONTRIBUTING.md file is not. It's all boiler plate and in many cases anyway.